README.md

```diff
@@ -130,6 +132,26 @@ jobs:
       - run: rebar3 ct
 ```
 
+### Basic example (`rebar3` on Windows 2016)
+
+```yaml
+# create this in .github/workflows/ci.yml
+on: push
+
+jobs:
+  test:
+    runs-on: windows-2019
+    name: Erlang/OTP ${{matrix.otp}} / rebar3 ${{matrix.rebar3}}
```

Comment:
this workflow doesn't specify a matrix, so `matrix.otp` etc is invalid here. I'd just leave this out from the snippet.

```suggestion
```

Reply:
Commited. Thanks.